ansible-scylla-node/handlers/main.yml

```diff
@@ -1,4 +1,11 @@
 ---
+
+- name: Clean yum metadata
+  command: yum clean metadata
+  args:
+    warn: no
+  become: true 
+
```

Comment:
why?
what problem does this solve? yum should be able to update itself, resp. we can force update before installing

Reply:
When removing a repository, beware that the metadata cache may still remain on disk until you run yum clean all. Use a notification handler for this.

_Source: https://docs.ansible.com/ansible/latest/collections/ansible/builtin/yum_repository_module.html_

The yum module does not support clearing yum cache in an idempotent way, so it was decided not to implement it, the only method is to use command and call the yum command directly, namely “command: yum clean all” https://github.com/ansible/ansible/pull/31450#issuecomment-352889579

_Source: https://docs.ansible.com/ansible/latest/collections/ansible/builtin/yum_module.html_

Reply:
I don't really know if the handler is the best way to do it. In any case, we can try to find a better way to achieve this.